src/config/runit/configuring.md

```diff
@@ -0,0 +1,44 @@
+# Configuring Services
+
+## Editing services
+
+To edit a service, first duplicate its service directory under a different name:
+
+```
+# cp -aR /etc/sv/service_name /etc/sv/service_name_edited
+```
+
+The contents of the duplicated directory can then be edited as needed, for
+example, by using `vi(1)`:
+
+```
+# vi /etc/sv/service_name_edited
+```
+
+After editing, the old service should then be stopped, disabled and replaced
+with the new one:
+
+```
+# sv down service_name
+# rm /var/service/service_name
+# ln -s /etc/sv/service_name_edited /var/service/
+```
+
+## Service directory layout
```

Comment:
I think this could go in the index page? (Or *maybe* an 'overview' page?) I like having this overview of how runit works, but I don't think it belongs in either of the two current subsections.

Reply:
Thanks for the feedback - PR updated.